src/ProjectTemplates/Web.ProjectTemplates/Grpc-ClientApplication.csproj.in

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
```

Comment:
This needs to be in the template?

Reply:
The template uses an `async Task Main` entry point in the console. It could be 7.1